tactile_state_publisher/setup.py

```diff
@@ -0,0 +1,9 @@
+from distutils.core import setup
```

Comment:
I won't change this, we build on melodic and this requires to change the package xml. in the noetic-devel in the future we would then drop distutils

Reply:
This change will be backwards compatible to Melodic. No need to hold it back.

Reply:
so you want me to update the package xml as well and upgrade the whole package.xlm to version 3 ?

Reply:
Yes, please. If we touch a project, we should take the chance to update it to become Noetic-compatible. If you like to do so in a separate PR, that's fine too.

Reply:
It will take some time to test if the change does not break installation. I will have to prepare a separate catkin_ws, totally independent from our distrib to ensure it is still found correctly. I will do that tomorrow then

Reply:
I (and many other ROS developers) are using this technique to achieve compatibility for both Melodic and Noetic.
There is no explicit need to test this technique.

Reply:
I never used this version of the package.xml so I might make a mistake. I want my PR to be tested.

Reply:
I updated tactile_toolbox to work with Noetic. Tested on both, Melodic and Noetic. Just rebase your branch(es) to HEAD.

Reply:
I am sorry I did not tackle as fast as you wished (my work day was already over). I did not try find excuses to make you do the job, I just was honest in telling I need a bit of time to read the REP and then apply the change, then setup a workspace that does install and test it. I was also worried that the build_generator would fail to read the new format of package.xml (due to renamed tags in version 2 and new tags/attributes in version 3) but had no local way to test this easily.

Reply:
No worries. I just wanted to follow your performance tests locally. Trying to do so, I noticed several other issues building on Noetic and fixed all of them.